tests/test_other.py

```diff
@@ -614,9 +614,15 @@ def test_emcc_cflags(self):
     self.assertContained('hello, world!', self.run_js('a.out.js'))
 
   def test_emcc_print_search_dirs(self):
-    result = self.run_process([EMCC, '-print-search-dirs'], stdout=PIPE, stderr=PIPE)
-    self.assertContained('programs: =', result.stdout)
-    self.assertContained('libraries: =', result.stdout)
+    output = self.run_process([EMCC, '-print-search-dirs'], stdout=PIPE).stdout
+    self.assertContained('programs: =', output)
+    self.assertContained('libraries: =', output)
+    self.assertContained(str(shared.Cache.get_lib_dir(absolute=True)), output)
+
+  def test_emcc_print_file_name(self):
+    self.run_process([EMBUILDER, 'build', 'libc'])
```

Comment:
`libc` is pretty slow to build. I guess on CI it will have already been built already. But perhaps just to avoid some future possible overhead, this could be something smaller?

Reply:
i.e. there is no practical situation where this actually makes a test run slower.... the only way you could imagine that would be that if you cleared you cache and then ran this one test in isolation.